website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -64,13 +64,20 @@ In this example we will use odo to manage a sample [Java JPA MicroService applic
     cd ./application-stack-samples/jpa
     ```
 
-3. Initialize the application:
+3. In case of IBM Z & Power, CRW images are used in devfiles ([Please refer this](https://odo.dev/docs/tutorials/deploying-a-devfile-using-odo-on-IBM-Z-and-Power)), Hence for [devfile.yaml](https://github.com/OpenLiberty/application-stack-samples/blob/main/devfiles/maven-image/devfile.yaml), replace `maven:3.6-adoptopenjdk-11-openj9` with `registry.redhat.io/codeready-workspaces/plugin-java11-openj9-rhel8`:
+    ```
+    sed -i "s/maven:3.6-adoptopenjdk-11-openj9/registry.redhat.io\/codeready-workspaces\/plugin-java11-openj9-rhel8/" ../devfiles/maven-image/devfile.yaml
+    cp ../devfiles/maven-image/devfile.yaml ./
+    odo create mysboproj
+    ```
+    
```

Comment:
Can you make this a subsection or note in the original point 3 i.e. `Initialize the application`?

Reply:
@valaparthvi great idea, what about I make this in original point 3: 
`3. Initialize the application on IBM Z & Power` ?

Reply:
I don't think that would be a good idea, since the tutorial is not specific to IBM Z & Power.  What would help in my opinion is having a special note for IBM Z & Power users under the original point 3.

Reply:
I am thinking something like the following - 
```md
3. Initialize the application:
    ```shell
    odo create java-openliberty mysboproj
    ```
   `java-openliberty` is the type of your application and `mysboproj` is the name of your application.

    **For IBM Z & Power systems:**
        3.1 cp ../devfiles/maven-image/devfile.yaml ./
        3.2 sed -i "s/maven:3.6-adoptopenjdk-11-openj9/registry.redhat.io\/codeready-workspaces\/plugin-java11-openj9-rhel8/" devfile.yaml
        3.3 odo create mysboproj
```